client/web/src/views/ChartViewContent/charts/line/components/LineChartContent.tsx

```diff
@@ -114,138 +131,272 @@ export function LineChartContent<Datum extends object>(props: LineChartContentPr
+                                    enableEvents={false}
+                                    xAccessor={accessors.x}
+                                    yAccessor={accessors.y[line.dataKey]}
+                                    // Don't have info about line in props. @visx/xychart doesn't expose this information
+                                    // Move this arrow function in separate component when API of GlyphSeries will be fixed.
+                                    /* eslint-disable-next-line react/jsx-no-bind */
+                                    renderGlyph={props => {
+                                        // Pay attention that props.key here is actually index of current datum.
+                                        // Implementation problem of @visx/xychart
+                                        const hovered =
+                                            hoveredDatum?.index === +props.key && hoveredDatum.key === line.dataKey
+                                        const focused =
+                                            focusedDatum?.index === +props.key && focusedDatum.key === line.dataKey
+
+                                        const linkURL = line.linkURLs?.[+props.key]
+                                        const currentDatum = {
+                                            key: line.dataKey.toString(),
+                                            index: +props.key,
+                                            datum: props.datum,
+                                        }
+
+                                        const xAxisValue = dateLabelFormatter(new Date(accessors.x(props.datum)))
+                                        const yAxisValue = (accessors.y?.[line.dataKey](props.datum) as string) ?? ''
+
+                                        return (
+                                            <MaybeLink
+                                                to={linkURL}
+                                                onPointerUp={stopPropagation}
+                                                onClick={onDatumLinkClick}
+                                                /* eslint-disable-next-line react/jsx-no-bind */
+                                                onFocus={() => linkURL && setFocusedDatum(currentDatum)}
+                                                /* eslint-disable-next-line react/jsx-no-bind */
+                                                onBlur={() => linkURL && setFocusedDatum(null)}
+                                                className="line-chart__glyph-link"
+                                                role={linkURL ? 'link' : 'graphics-dataunit'}
+                                                aria-label={`Point № ${+props.key + 1} of line ${index + 1} of ${
+                                                    series.length
+                                                }. X value ${xAxisValue}. Y value ${yAxisValue}`}
```

Comment:
```suggestion
                                                aria-label={`Point ${+props.key + 1} of line ${index + 1} of ${
                                                    series.length
                                                }. X value: ${xAxisValue}. Y value:  ${yAxisValue}`}
```
Do we have access to the series name here to use instead of a line index? Or maybe we could just nest the points under the series group, so it doesn't need to be repeated but is contextual.

Reply:
My origin though on that was - we can't put `GlyphSeries` inside `LineGroup` because of our custom rect element should place under GlyphSeries but over LineGroup. But right now it seems like maybe we can put this rect under both of these and have only one group for each series. I'll try that